### ISSUE TITLE
add thumbnail/graphicoverview to the core record model

### ISSUE THREAD

**Issue by pvgenuchten** [opengeospatial/ogcapi-records#94]
A suggestion to add thumbnail (graphicoverview in iso19139) to the core model as optional field.
Thumbnail is quite common in for example https://schema.org/thumbnailUrl and [opengraph](https://ogp.me)

**Comment by pvgenuchten** [comment]
What kind of feedback is needed?

Having a thumbnail for records is a common use case. The thumbnail is displayed in search results:

![image](https://user-images.githubusercontent.com/299829/109285032-d99a8080-7820-11eb-8c20-7a414652a57d.png)

But it can also be used to add an opengraph:image to the html representation, so the thumbnail is shown when you include a link to the record in social media

![image](https://user-images.githubusercontent.com/299829/109285153-00f14d80-7821-11eb-8ebf-f93a158c1964.png)

**Comment by pvretano** [comment]
@pvgenuchten we are waiting to hear back from more implementors.  That will take time.
As far as I can recall, thumbnail was never a CSW core queryable either. Don't get me wrong, I use thumbnails too (see: https://www.pvretano.com/cubewerx/cubeserv/default/ogcapi/catalogues/collections/sentinel1cat/items) but its not like you can use thumbnail in a query expression.  Of course, you are not prevented from including it as a record property in your community of interest.

**Comment by pvgenuchten** [comment]
I agree that thumbnail is not supposed to be a queryable (maybe as true/false, e.g. hasThumbnail)
But I would be interested if everybody uses a similar key for it, and not a mix of thumbnail, thumbnailUri, image, graphicOverview, etc...

**Comment by MichaelGordon** [comment]
From Ordnance Survey's point of view, we will usually use a thumbnail for our records, so this would definitely be useful to include as an optional attribute in records.

**Comment by m-mohr** [comment]
In STAC, this is part of the assets. As there are no assets here, the next logical place for thumbnails would simply be a link with a pre-defined rel type. For example:
```
{
  href: 'https://example.com/logo.png',
  rel: 'thumbnail',
  type: 'image/png'
}
```

**Comment by pvretano** [comment]
19-APR-2021: Add a recommendation to the specification for this particular use case since it has come up a few times.  The only difference is that we would use the link relation 'preview' rather than thumbnail since the former is an official IANA link relation.  See: https://tools.ietf.org/html/rfc6903#section-3

**Comment by m-mohr** [comment]
Yes, that sounds good to me. Indeed, thumbnail (and overview) are just the roles in STAC. I think STAC should also mention preview as rel type though, it's the more general way of adding thumbnails in comparison to assets.

**Comment by m-mohr** [comment]
PR for STAC: https://github.com/radiantearth/stac-spec/pull/1092

**Comment by tomkralidis** [comment]
I think thumbail / graphicoverview is pertinent and valuable from a MetOcean perspective as well (as a returnable).  +1 to express as a link with `rel: preview`.

**Comment by tomkralidis** [comment]
Discussed as the [July 2021 OGC API Virtual Code Sprint](https://github.com/opengeospatial/ogcapi-code-sprint-2021-07), will move this to the Part 2 (dataset record) once proposed.

**Comment by pvretano** [comment]
09-AUG-2021: This issue can be closed as it will become part of #130.  Basically, in #130 there will be a link related of `preview` to point to a thumbnail.  So, the thumbnail will not be part of the core.
